pie/utils/objects.py

```diff
@@ -12,19 +10,43 @@
 _ = i18n.Translator("pie").translate
 
 
-class ScrollableEmbed:
+class ScrollableEmbed(nextcord.ui.View):
     """Class for making scrollable embeds easy.
 
     Args:
         ctx (:class:`nextcord.ext.commands.Context`): The context for translational purposes.
         iterable (:class:`Iterable[nextcord.Embed]`): Iterable which to build the ScrollableEmbed from.
+        timeout (:class:'int'): Timeout in seconds (default 300)
+        delete_after (:class:'bool'): Delete message after timeout (default False)
```

Comment:
I don't know about the naming here. What I would expect seeing this attribute name would be a duration after which the message is deleted. 

We can keep it this way but I would prefer a clearer name.

Otherwise this looks really nice

Reply:
Ok, I'll try to think about it.

Reply:
I made docs change and also renamed those attributes.

And reverted changes I accidentaly made in ConfirmView (not including docs fix).